packages/material-ui-based-components/__tests__/components/TabsVerticalComp.test.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2023 Adobe, Inc.
+ *
+ * Your access and use of this software is governed by the Adobe Customer Feedback Program Terms and Conditions or other Beta License Agreement signed by your employer and Adobe, Inc.. This software is NOT open source and may not be used without one of the foregoing licenses. Even with a foregoing license, your access and use of this file is limited to the earlier of (a) 180 days, (b) general availability of the product(s) which utilize this software (i.e. AEM Forms), (c) January 1, 2023, (d) Adobe providing notice to you that you may no longer use the software or that your beta trial has otherwise ended.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL ADOBE NOR ITS THIRD PARTY PROVIDERS AND PARTNERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import TabsHorizontalComp from '../../src/components/TabsHorizontalComp'
+import React from "react";
+import { render } from "@testing-library/react";
+import { renderComponent } from "../utils";
+import "@testing-library/jest-dom/extend-expect"
+import userEvent from "@testing-library/user-event";
+
+const emptyTabs = {
+  id: "emptyTabs",
+  ":type": "core/fd/components/form/verticaltabs/v1/verticaltabs",
+  visible: true,
+  label: { value: "empty label" },
+  items: [],
+  index: 0,
+};
+
+const tabsWithData = {
+  id: "tabsWithData",
+  ":type": "core/fd/components/form/verticaltabs/v1/verticaltabs",
+  visible: true,
+  label: { value: "tab label" },
+  index: 0,
+  items: [
+    {
+      id: "field",
+      name: "f1",
+      fieldType: "text-field",
+      visible: true,
+      index: 0,
+      label: { value: "text field label" },
+    },
+    {
+      id: "field2",
+      name: "f2",
+      fieldType: "text-field",
+      visible: true,
+      index: 1,
+      label: { value: "text field 2 label" },
+    },
+  ],
+};
+
+const helper = renderComponent(TabsHorizontalComp);
```

Comment:
The test cases are using the horizontal tabs component instead of verticalTabs comp please change that for all the test cases.